ultra/ultra/scenarios/task1/config.yaml

```diff
@@ -103,27 +97,15 @@ levels:
         end_offset:   [10, 10]
       intersection_types:
         2lane_c:
-          percent: 0.2
-          specs: [[50kmh,no-traffic,0.14],[70kmh,no-traffic,0.13],[100kmh,no-traffic,0.13], #40%
-                  [50kmh,low-interaction,0.10],[70kmh,low-interaction,0.10],[100kmh,low-interaction,0.10], #30%,
-                  [50kmh,mid-interaction,0.07],[70kmh,mid-interaction,0.065],[100kmh,mid-interaction,0.065], #20% 
-                  [50kmh,high-interaction,0.04],[70kmh,high-interaction,0.03],[100kmh,high-interaction,0.03]] #10%
+          percent: 0.5
+          specs: [[50kmh,no-traffic,0.20],[70kmh,no-traffic,0.20],[100kmh,no-traffic,0.20], #60%
+                  [50kmh,low-interaction,0.14],[70kmh,low-interaction,0.13],[100kmh,low-interaction,0.13]] #40%
```

Comment:
I recall there were talks about trying not to add new distributions to the existing tasks' levels. Have these concerns been resolved?

Reply:
@christianjans You are correct, I actually have implement a new simple level in the curriculum learning branch, but those changes have not carried over to this branch. I will change them right away

Reply:
Refer to 74f46b3